packages/terra-menu/src/orion-fusion-theme/Menu.module.scss

```diff
@@ -11,6 +11,7 @@
     --terra-menu-submenu-header-background-color: #f4f4f4;
     --terra-menu-submenu-header-border-bottom: 2px solid #9b9fa1;
     --terra-menu-submenu-header-button-outline-offset: -4px;
+    --terra-menu-submenu-header-button-outline: #004c76 solid 2px;
```

Comment:
this variable is not present in Menu.module.scss and MenuItem.module.scss. why do we on theme files ..?

Reply:
Updated style file - https://github.com/cerner/terra-framework/pull/1801/commits/33e14a6220d3246fca3812a66d8090645a1b56f9